trieste/utils/mo_objectives.py

```diff
@@ -0,0 +1,161 @@
+# Copyright 2020 The Trieste Contributors
```

Comment:
Might it be clearer to name the file multi_objectives.py?

Reply:
Yes, changed